tests/python_on_whales/components/test_system.py

```diff
@@ -50,7 +50,7 @@ def test_prune_prunes_image():
     assert image in docker.image.list()
 
     # image not pruned because it does not have dne label
-    docker.system.prune(all=True, filters={"label": "dne"})
+    docker.system.prune(all=True, filters={"label": "dne", "until": "1000000h"})

```

Comment:
Could you put that in another test (you may copy paste the code), so that we have a test with one filter and a test with two filters.

Reply:
I will put this all in one test function because that seems a bit cleaner. Let me know what you think.